examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1192 @@
+    "\n",
+    "In all the simulations we employ $\\nu=0.4 $. In this way, the temporal parameters are given by:\n",
+    "\n",
+    "- $\\Delta t$ $\\approx$ 0.0016 s = 1.6 ms;\n",
+    "- $NT$ = 626.\n",
+    "\n",
+    " The source $f(x, z, t)$  will be a Ricker source with the following properties:\n",
+    "\n",
+    "- Postion at $x:$ $\\bar{x} = 500 m = 0.5 Km$;\n",
+    "- Position at $z:$ $\\bar{z} = 10 m = 0.01 Km$;\n",
+    "- Peak frequence: $f_{0} = 10 Hz = 0.01 Khz$;\n",
+    "\n",
+    "The graph of $f(\\bar{x}, \\bar{z}, t)$ will be generated when building the code. We employ a synthetic velocity model $c(x,z)$ with the following properties:\n",
+    "\n",
+    "- Minimum propagation speed: $v_{min} = 1500 m/s = 1,5 Km/s$;\n",
+    "- Maximum propagation speed: $v_{max} = 2500 m/s = 2,5 Km/s$;\n",
+    "\n",
+    "The figure of the velocity profile will be generated when building the code. We will consider 'receivers' along the $x$ direction (at all discrete points between $0.0$ Km and $1.0$ Km) at the depth $z=0.01$ Km, where we measure the displacement $u(x,z,t)$ throughout the time integration. Based on these values at the receivers we bild a $seismogram$ at the end of the integration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1.5 - Numerical Simulations\n",
+    "\n",
+    "In the numerical simulations, we initially import the packages of interest, in particular, those that are part of the Devito library. First we import the basic Python packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy                   as np\n",
+    "import matplotlib.pyplot       as plot\n",
+    "import math                    as mt\n",
+    "import matplotlib.ticker       as mticker    \n",
+    "from   mpl_toolkits.axes_grid1 import make_axes_locatable\n",
+    "from   matplotlib              import cm\n",
+    "import time                    as tm\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Devito's library of examples, we import some specific functions, such as:\n",
+    "\n",
+    "- *TimeAxis* which generates a temporal structure;\n",
+    "- *RickerSource* for the setup of the source term;\n",
+    "- *Receiver* which generates a structure of receivers;\n",
+    "\n",
+    "So, we import these sample libraries and other importants libraries with the following commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from   examples.seismic  import TimeAxis\n",
+    "from   examples.seismic  import RickerSource\n",
+    "from   examples.seismic  import Receiver\n",
+    "from   devito            import SubDomain, Grid, NODE, TimeFunction, Function, Eq, solve, Operator, configuration\n",
+    "configuration['log-level']='PERF'"
```

Comment:
It would be worth adding a comment re. this.

Reply:
I have already added a comment about the configuration used.